hil/ext/switches/ovs.py

```diff
@@ -79,11 +77,12 @@ def get_port_networks(self, ports):
 
     def revert_port(self, port):
 
-        shell_cmd_1 = 'sudo ovs-vsctl del-port {port}'.format(port=port)
-        shell_cmd_2 = 'sudo ovs-vsctl add-port {switch} {port}'.format(
-                    switch=self.ovs_bridge, port=port
-                    ) + ' vlan_mode=native-untagged'
-        self.ovs_connect(shell_cmd_1, shell_cmd_2)
+        args_1 = ['sudo', 'ovs-vsctl', 'del-port', str(port)]
+        args_2 = [
+                'sudo', 'ovs-vsctl', 'add-port', str(self.ovs_bridge),
+                str(port), 'vlan_mode=native-untagged'
+                ]
+        self.ovs_connect(args_1, args_2)
```

Comment:
No reason to introduce intermediate variables:

```python
self.ovs_connect(
    ['sudo', 'ovs-ctl', 'del-port', ...],
    ['sudo', ...],
)
```

You've got a few other places where you create an unnecessary intermediate variable as well.

Reply:
I have added the extra variable assignment for readability and keep the style consistent where I have to struggle to keep within 80 chars per line. 
I strongly feel that this is more readable than the suggested way of writing besides one extra variable assignment is not going to hurt the performance.

Reply:
I don't think this would increase line length? The lists would be less indented than they are now.

Performance has nothing to do with it. I disagree that separate assignments are more readable here, but don't feel strongly enough to block on it.

Tangentially related: I'm wondering if maybe we should increase the hard line-length limit to 100, and have 80 be a soft recommendation? I occasionally find myself having to be creative to fit stuff in 80 chars as well. @naved001, @Izhmash, thoughts?

Reply:
> Tangentially related: I'm wondering if maybe we should increase the hard line-length limit to 100, and have 80 be a soft recommendation

I am fine with that.

Reply:
I think even pep8 docs are okay with having a limit at 100 characters for code, but still keeping the length of docstrings to be 72 (or 79).